sdk/keyvault/Azure.Security.KeyVault.Administration/tests/AccessControlClientLiveTests.cs

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using Azure.Security.KeyVault.Administration.Models;
+using static Azure.Security.KeyVault.Administration.KeyVaultAdministrationConstants;
+using NUnit.Framework;
+using System.Linq;
+
+namespace Azure.Security.KeyVault.Administration.Tests
+{
+    public class AccessControlClientLiveTests : AccessControlTestBase
+    {
+        private const string pricipalID = "4ae6842b-2a3e-4919-8305-7db89f3d6edd";
```

Comment:
Do not hardcode values for identifying material. These must come from the environment, e.g. test-resources.json. I can help you with that if you need, but see our `KeyVAultTestEnvironment` for an example of how to get them properly, and you can create roles (if needs be) and query for them via ARM like I have in sdk\search\test-resources.json (example of getting an ARM-created storage key, but the same applies with slight modification for role IDs). You can probably even just use the current principal ID unless you explicitly need a separate one.